Source/Immutable/Private/Immutable/ImtblBlui.cpp

```diff
@@ -60,9 +60,9 @@ void UImtblBlui::BeginDestroy() {
   IMTBL_LOG_FUNCSIG
 #if USING_BLUI_CEF
   if (GetBluEye()) {
-    GetBluEye()->CloseBrowser();
+    // GetBluEye()->CloseBrowser();
```

Comment:
Should we remove it completely? same for the one below?

Reply:
We need it. I reinstated it in the code style commit.